<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QStackedWidget" name="pageWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="SettingsTab" name="settingsPage"/>
   <widget class="AccountTab" name="accountsPage"/>
   <widget class="LogTab" name="logPage"/>
   <widget class="AboutTab" name="aboutPage"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSettings"/>
   <addaction name="actionAccount"/>
   <addaction name="actionLog"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionSettings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/switch.png</normaloff>:/icons/switch.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/log@2x.png</normaloff>:/icons/log@2x.png</iconset>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/info@2x.png</normaloff>:/icons/info@2x.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
  <action name="actionAccount">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/person@2x.png</normaloff>:/icons/person@2x.png</iconset>
   </property>
   <property name="text">
    <string>Account</string>
   </property>
   <property name="toolTip">
    <string>Account</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SettingsTab</class>
   <extends>QWidget</extends>
   <header>src/ui/SettingsTab.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LogTab</class>
   <extends>QWidget</extends>
   <header>src/ui/LogTab.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AboutTab</class>
   <extends>QWidget</extends>
   <header>src/ui/AboutTab.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AccountTab</class>
   <extends>QWidget</extends>
   <header>src/ui/AccountTab.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
